### PR TITLE
Added Open Web API Link

### LINKED ISSUE #112: Button to open the Record in the WebAPI
One thing I find myself using the tool for frequently is to copy the ID of the record specifically so that I can open it up in the web API. 

It would be nice if there was a button to open the record in the Web API within a separate tab/window

### PR DESCRIPTION
**Pull request by pcpinme**
# :clipboard: **Description**

Updated NPM dependencies to latest versions.
Added Web API link. #112 
 
# :white_check_mark: **Code Review Checklist**

- [x] I completed a complete self-review of my contribution.

I tested the extension for the following browsers:
  - [x] Google Chrome
  - [x] Mozilla Firefox
  - [x] Microsoft Edge
  - [x] Microsoft Edge (Chromium)

I tested the extension on the following versions of the product:
 - [x] Dynamics 365 v9
 - [x] Dynamics 365 v8.2
 - [x] Dynamics CRM 2016
 - [x] Dynamics CRM 2015
 - [x] Dynamics CRM 2013

### CODE REVIEW

File: src/js/ui/pane.js [hunk 1]
```diff
@@ -1205,6 +1205,27 @@ $(function () {
...
+							var url = Xrm.Page.context.getClientUrl() + "/api/data/v" + shortVersion + "/" + result.EntitySetName +"(" + Xrm.Page.data.entity.getId() + ")";
+							url = url.replace("{", "").replace("}","");
+					
+							window.open(url,'_blank');
+							
+						});
+                    
+                } catch (e) {
+                    CrmPowerPane.UI.ShowNotification("An error occured opening the Web API URL for this record.");
+                }
```
Comment (mehmetseckin) [inline src/js/ui/pane.js:1227]: Not sure if it's an editor/browser issue, but the indentation seems a bit off on the browser.

File: src/html/ui/pane.html [hunk 2]
```diff
@@ -46,6 +46,7 @@
                 <li class="crm-power-pane-subgroup" id="crm-diagnostics"><span><span class="icon crm-diagnostics-icon"></span>Crm Diagnostics</span></li>
                 <li class="crm-power-pane-subgroup" id="performance-center"><span><span class="icon performance-center-icon"></span>Performance Center</span></li>
                 <li class="crm-power-pane-subgroup" id="mobile-client"><span><span class="icon mobile-icon"></span>Mobile Client</span></li>
+				<li class="crm-power-pane-subgroup" id="open-webapi"><span><span class="icon record-url-icon"></span>Open Web API Record</span></li>
```
Comment (mehmetseckin) [inline src/html/ui/pane.html:49]: Again, the indentation seems a bit off on the browser, not sure if it's due to editor preferences.

**Review by mehmetseckin — CHANGES_REQUESTED** [review]
Wow, thanks a lot for your contribution! This looks good, very nice work! The indentation seemed a little off in a couple of places, not sure if it's a browser/editor issue. Apart from that, awesome job! :shipit:

File: src/html/ui/pane.html (hunk 2 above)
Comment (pcpinme) [reply src/html/ui/pane.html]: Had to fix using Notepad. Not sure what Notepad++ or VS Code is showing correctly. Thanks for the feedback.

**Review by mehmetseckin — APPROVED** [review]

**Review by onurmenal — APPROVED** [review]
Thank you so much for your contribution. It is great work!✨

**Comment by pcpinme** [comment]
Fixed indentation properly and also extended the Navigation section across two columns.

**Comment by pcpinme** [comment]
When does this get merged back?

**Comment by onurmenal** [comment]
> When does this get merged back?

Hi Paul,  It will be tested and merged this week. If it passes tests, I will start the publishing process.

**Comment by pcpinme** [comment]
Hi @onurmenal  , what is the process to have this feature available in the production version in the Chrome web store?

**Pull request merged**